community/i3-gaps/build

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh -e
+
+# export V=1 for verbose output when building
+[ "${V:-0}" -eq 1 ] > /dev/null 2>&1 && set -x || :
+
+export CHOST="${CHOST:-x86_64-pc-linux-musl}"
+
+PATCHES="${PATCHES:-}"
+
+PATCHES="
+i3-4.16-musl-GLOB_TILDE.patch
+${PATCHES}
+"
+
+if [ -n "${PATCHES}" ]
+then
+	for patch in ${PATCHES}
+	do
+		if [ -z "${patch}" ] || [ ! -e "${patch}" ]
+		then
+			continue
+		fi
+		patch -p1 < "${patch}"
+	done
+fi
+
+autoreconf -fvi
+
+./configure --prefix=/usr --enable-sanitizers
```

Comment:
Move each flag to a new line indented by 4 spaces.

Is `--enable-sanitizers` needed?

Reply:
Sanitizers shouldn't be needed but it could help find issues (Arch enables it by default). As a compromise I'll leave this unset by default but add an `$EXTRA_I3_CONF` variable (`./configure --prefix=/usr ${EXTRA_I3_CONF}` so users can pass extra options to configure if they want to (Gentoo does this for some packages and it basically allows you to do something like `env EXTRA_I3_CONF="--enable-sanitizers" kiss build i3-gaps` and then your variable gets expanded with any extra configuration options you wanted, you'd of course have to put that variable in a config file somewhere or wrapper script to make it persist).

Reply:
We don't currently add any package specific environment variables and I really don't want to start. The intended behavior for package customization is a user repository, however we're possibly exploring a `USE_FLAGS` kind of system for the future.